tests/setup.ts

```diff
@@ -0,0 +1,34 @@
+import * as dotenv from 'dotenv';
+import { chromium } from '@playwright/test';
+import type { FullConfig, Browser } from '@playwright/test';
+
+dotenv.config();
+
+async function globalSetup(config: FullConfig) {
+    const { storageState } = config.projects[0].use;
+
+    const chromiumBrowser = await chromium.launch();
+    await setupPage(chromiumBrowser, storageState as string)
+}
+
+async function setupPage(browser: Browser, storageState: string) {
+    const page = await browser.newPage();
+    await page.goto(`${process.env.APPLICATION_BASE_URL}/logto/sign-in`);
+    await page.waitForURL(`${process.env.LOGTO_ENDPOINT}/sign-in`, {
+		waitUntil: 'domcontentloaded',
+	});
+
+    await page.getByPlaceholder('Email / Username').fill(process.env.TEST_USER_EMAIL);
+    await page.getByRole('button', { name: 'Sign in' }).click();
+    await page.getByPlaceholder('Password').fill(process.env.TEST_USER_PASSWORD);
+    await page.getByRole('button', { name: 'Continue' }).click();
+    await page.waitForURL(`${process.env.APPLICATION_BASE_URL}/swagger/`, {
+        waitUntil: 'domcontentloaded',
+        timeout: 60000,
+    });
+
+    // End of authentication steps.
+	await page.context().storageState({ path: storageState });
+}
+
+export default globalSetup;
```

Comment:
Besides logging in, what might be other things we could do in global setup? Some examples I can think of are:

1. Once you're logged in, check if GET account returns customer ID and a cheqd address. (The button might have changed after logging in, but that's no guarantee that you're _actually_ logged in until you can get an account response.) Once we expand the DB table for customers, you could probably check for other elements.

I have a few other ideas, but tbh they require changes to the Account & Key APIs **so we should skip these for now** but other things could have been:
1. IMO, the GET account API could/should also return the account balance in CHEQ, which would allow us in global setup to test whether there's sufficient balance.
2. If you actually click the Account create API, it throws an error because there's already an associated account. The original intention of course was to create new Cosmos/cheqd accounts under the same Customer ID.
3. Key create is also kinda meaningless, since we always generate new keys for every DID create request (for example) and don't allow people to specify a key alias to use. If we actually supported specifying an existing key, then perhaps one of the global setup steps would have been to create a new key before the execution of further tests (and then using that key in a DID create).